update/common.go

```diff
@@ -0,0 +1,198 @@
+package update
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/rewardTx"
+	"github.com/ElrondNetwork/elrond-go/data/smartContractResult"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/data/state/factory"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+)
+
+const MetaBlockFileName = "metaBlock"
+const TransactionsFileName = "transactions"
+const MiniBlocksFileName = "miniBlocks"
+
+// Type identifies the type of the export / import
+type Type uint8
+
+const (
+	Transaction         Type = 0
```

Comment:
done.

Reply:
I would do it like this:

```go
const (
        Unknown Type = iota
	Transaction
	SmartContractResult
	RewardTransaction
	MiniBlock
	Header
	MetaHeader
	AccountsTrie
	RootHash
	UserAccount
	PeerAccount
	MetaAccount
)
```

The number will be auto incremented and this way you can change the order without affecting functionality. I moves `Unknown` first because if someone forgets to initialise a `Type` ex: `var t Type`, t will be unknown by default

Reply:
done.